indico/web/assets/vars_js.py

```diff
@@ -84,6 +84,7 @@ def generate_user_file(user=None):
             'address': user.address,
             'phone': user.phone,
             'type': user.principal_type.name,  # always 'user'
+            'mastodonServerURL': user.settings.get('mastodon_server_url') or None,
```

Comment:
`or None` makes me thing you allow saving empty strings. I'd rather send a `null` in the form, and disallow empty strings on the schema, so it's either a url or None/null.

Alternatively, disallow `null` and use an empty string as the default.